puppet/modules/freight/manifests/user.pp

```diff
@@ -70,6 +70,8 @@
   include apache::mod::mime
 
   web::vhost { $vhost:
+    servername    => "${vhost}.${facts['networking']['fqdn']}",
```

Comment:
I have created `*.web01.osuosl` and `*.repo-deb01.osuosl` in DNS.

Reply:
I'd prefer `-backend`, even though you're right it probably doesn't matter.

Now a question is: should it be the servername or an alias.  I have a vague recollection that the servername is used in built in error pages. https://httpd.apache.org/docs/2.4/mod/core.html#servername at least suggests that it should be the name end users connect to.

Reply:
But the long term idea is not to ever have the name the user connects to in this config.

And we (should) use ServwrSignature Off (at least right now we do), so there is no footer and anything

Reply:
https://www.fastly.com/documentation/guides/integrations/backends/#overriding-the-host-header states:

> If you use a hostname (rather than an IP address) to define your backend, Fastly will only use the hostname to look up the IP address of the server, not to set the [Host](https://www.fastly.com/documentation/reference/http/http-headers/Host/) header or negotiate a secure connection. By default the [Host](https://www.fastly.com/documentation/reference/http/http-headers/Host/) header on backend requests is copied from the client request. For example, if you own www.example.com and point it to Fastly, and create a second domain of origin.example.com that points to your origin, then the web server running on your origin and serving origin.example.com must also be able to serve requests that have a Host: www.example.com header. Fastly will also use the client-forwarded hostname to establish a secure connection using [Server Name Indication](https://en.wikipedia.org/wiki/Server_Name_Indication).

So we need to explicitly configure it in a different way. Are we doing that?

Reply:
Today, we set `address`:
https://github.com/theforeman/foreman-infra/blob/85293f48404e0ddcafbad0e982d9c5a8a49f24c7/ansible/fastly.yml#L17
And `ssl_*`:
https://github.com/theforeman/foreman-infra/blob/85293f48404e0ddcafbad0e982d9c5a8a49f24c7/ansible/fastly.yml#L20-L21
So we'd have to set `override_host` too.

Reply:
https://github.com/theforeman/foreman-infra/pull/2153